week-1/F-strings-concatenation/exercise.js

```diff
@@ -1,3 +1,6 @@
 // Start by creating a variable `message`
+//program that logs a message with a greeting and your name
+var greetings = 'Hello, my name is ';
+var myName = 'Jordan';
 
-console.log(message);
+console.log(greetings + myName);
```

Comment:
Very good. Note how in this case the `+` operator has a different meaning from when you use it with numbers.